lib/components/tiptap/floatingMenu/floatingMenuSuggestion.tsx

```diff
@@ -1,72 +1,72 @@
-import { Editor, ReactRenderer, Range } from '@tiptap/react'
-import tippy from 'tippy.js'
+import { Editor, ReactRenderer, Range } from "@tiptap/react";
+import tippy from "tippy.js";
 
-import { FloatingMenu } from './FloatingMenu'
-import { TiptapEditorUtils } from './../../../utils/tiptapEditorUtils'
+import { FloatingMenu } from "./FloatingMenu";
+import { TiptapEditorUtils } from "./../../../utils/tiptapEditorUtils";
 
 export const floatingMenuSuggestion = {
   items: ({ query }: any) => {
-    const normalizedQuery = query.toLowerCase().replace(' ', '');
+    const normalizedQuery = query.toLowerCase().replace(" ", "");
     return [
       {
-        title: 'Heading 1',
+        title: "Heading 1",
         command: ({ editor, range }: any) => {
-          const tiptapEditorUtils = new TiptapEditorUtils(editor)
-          tiptapEditorUtils.deleteRange(range)
-          tiptapEditorUtils.toggleHeading(1)
+          const tiptapEditorUtils = new TiptapEditorUtils(editor);
+          tiptapEditorUtils.deleteRange(range);
+          tiptapEditorUtils.toggleHeading(1);
         },
       },
       {
-        title: 'Heading 2',
+        title: "Heading 2",
         command: ({ editor, range }: { editor: Editor; range: Range }) => {
-          const tiptapEditorUtils = new TiptapEditorUtils(editor)
-          tiptapEditorUtils.deleteRange(range)
-          tiptapEditorUtils.toggleHeading(2)
+          const tiptapEditorUtils = new TiptapEditorUtils(editor);
+          tiptapEditorUtils.deleteRange(range);
+          tiptapEditorUtils.toggleHeading(2);
         },
       },
       {
-        title: 'Heading 3',
+        title: "Heading 3",
```

Comment:
Change prettier to prefer single quotes instead of double... this is causing a lot of unnecessary changes